tests/improved-sanity-test/main.yml

```diff
@@ -210,6 +210,24 @@
       tags:
         - atomic_host_check
 
+    # we remove any subscriptions after the upgrade to verify that
+    # 'rpm-ostree status' with the 'unconfigured-state' field present.
+    # https://bugzilla.redhat.com/show_bug.cgi?id=1421867
+    - role: redhat_unsubscribe
+      unconfigured_state: true
+      when: ansible_distribution == 'RedHat'
+      tags:
+        - redhat_unsubscribe
+
```

Comment:
I would make sure to stop the daemon here before running `status`. Though from looking at the tasks above it seems like it shouldn't be running yet anyway, but it's not obvious either that we rely on that here. So I can see that assumption easily becoming false in the future.